findspam.py

```diff
@@ -150,273 +156,280 @@ def keyword_email(s, site):   # a keyword and an email in the same post
-                            "softaken\\.com", "lyonstechnologies", "serialkeygeneratorfree\\.com", "routeperfect\\.com",
-                            "tupely\\.com", "apk(heart|safe)\\.com", "uflysoft\\.net", "nimblemessaging\\.com",
-                            "oleville\\.net", "nutribulletrecipes\\.org", "wirexapp\\.com", "x4up\\.org",
-                            "decalontop\\.com", "urlopener\\.com", "mobile57\\.com", "learn(spicy|perfact)",
-                            "getfitness\\.in", "trustwiko\\.com", "attendasoft", "selfybuzz\\.com", "meritcampus\\.com",
-                            "fastindiaservice\\.com", "shharshsajv", "fizyetimusing", "fornatgaex", "shwesanenid",
-                            "accountingassignments\\.help", "phphelponline\\.com", "eremaxfuncionabr", "zu-rich\\.li",
-                            "appsapkfile\\.com", "bandarterbaikterpercaya\\.net", "yourdailymovie\\.com",
-                            "ipinteria\\.com", "blogines\\.com", "stepupheights\\.com", "gfix\\.in",
-                            "aminoprimexl\\.com", "csharpstar\\.com", "vbscore\\.com", "blueeagleindia\\.com",
-                            "vizayn\\.com", "health(flyup|buzzer)\\.com", "androidpureapk\\.com",
-                            "upsafe\\.com", "spiritsofts\\.com", "rcptec\\.com", "gmax-brasil\\.com", "icognix\\.net",
-                            "\\Wpysoft\\.com", "zescode\\.com", "eserviceshelp\\.in", "captainform\\.com",
-                            "techiphone\\.com", "kmminoaq4yci5woj\\.onion", "BlackListHackers\\.com",
-                            "transferphone\\.com", "hindipathshala\\.com", "applify\\.co", "armmlm\\.com",
-                            "snipercrack\\.tk", "averagemaleheight\\.tk", "educba\\.com", "neosurftobitcoin\\.net",
-                            "silver-card\\.net", "cards101\\.net", "hakerstars\\.com", "king-dumps\\.us",
-                            "cuidados-saude", "klereumcol\\.com", "gupshupchatroom\\.com", "petsworld\\.in",
-                            "godiabetesrevenge", "reflectivevestsindia", "anyframe\\.net", "canadianprofits\\.tv",
-                            "dcweddingandevents\\.com", "slimdreneavis", "wefix365\\.us", "esofttools\\.com",
-                            "wondershare\\.com", "pulsionerotica\\.com", "worldtraveltime\\.net", "antivirus\\.comodo\\.com",
-                            "cardvdonline\\.com", "icasnetwork\\.org", "epicresearch\\.co", "\\.soup\\.io", "pccdkeys\\.com",
-                            "hotxt\\.co\\.uk", "rcframecontractors", "bsgolds\\.com", "okaygoods\\.com", "thedropnet\\.com",
-                            "science\\.misis\\.ru", "pdf\\.ac/", "slotobit\\.com", "thaykinhmanhinhcamung\\.com",
-                            "sfdcbot\\.herokuapp\\.com", "technosanatgroup\\.com", "csgostrong\\.com", "freenom\\.com"]
+    bad_keywords = [
+        "baba ji", "fifa.{0,20}coins?", "fifabay", "Long Path Tool",
```

Comment:
It reduces the indentation, I suppose. I don't mind for smaller blocks, but for massive blocks it makes a bit of a difference